docs/development/release.md

```diff
@@ -12,11 +12,14 @@ Read more about the semantic versioning [here](https://docs.npmjs.com/getting-st
 
 #### From Master
 
-If creating a new minor version release from master, create a branch titled `release-MAJOR.MINOR.x` (e.g. release-1.14.x) as a minor version branch from master to allow us to easily backport patches to that version.  Then create a release branch from the minor version branch.
+If creating a new minor version release from master, create a branch titled `release-MAJOR.MINOR.x` (e.g. release-1.14.x) as a minor version base branch from master to allow us to easily backport patches to that version.
+
+Push the `release-MAJOR.MINOR.x` branch to GitHub. (Commits should be even with master at this point.)
```

Comment:
It wasn't clear to me until step 6 that I should have basically done this here.

Reply:
Hmm, fair enough.  Remember that you can create a branch on github as well, so you might not have to push.      This was deliberately vague, but I think this is probably fine.

Reply:
beedf6deb